### PR TITLE
DOC-4892: Added clarification points on AWS DMS docs

### PR DESCRIPTION
**Pull request by nickvigilante**
Fixes DOC-4892
Fixes DOC-4968

### CODE REVIEW

**Review by awoods187 — APPROVED** [review]
> Reviewed 2 of 2 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14482)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ajstorm, @devadvocado, and @nickvigilante)

___
*[`v22.1/aws-dms.md` line 26 at r1](https://reviewable.io/reviews/cockroachdb/docs/14482#-N6PXAphB_fR6B9b45lb:-N6PXAphB_fR6B9b45lc:bsjqw36) ([raw file](https://github.com/cockroachdb/docs/blob/2ed345490151cc466fda0b7de53b19249ea5fdb5/v22.1/aws-dms.md#L26)):*
> ```Markdown
> - Configure a [source endpoint](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.html) in AWS pointing to your source database.
> - Configure a [replication instance](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_ReplicationInstance.html) in AWS.
> - Ensure you have a secure, publicly available [{{ site.data.products.dedicated }} cluster](../cockroachcloud/quickstart-trial-cluster.html) or [{{ site.data.products.core }} cluster] running v22.1 or higher.
> ```

isn't this 21.1.1? Or was the .0 sufficient?

**Review by nickvigilante — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14482)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ajstorm, @awoods187, and @devadvocado)

___
*[`v22.1/aws-dms.md` line 26 at r1](https://reviewable.io/reviews/cockroachdb/docs/14482#-N6PXAphB_fR6B9b45lb:-N6Pa2TX2eICHV_ZJZs3:b8trf45) ([raw file](https://github.com/cockroachdb/docs/blob/2ed345490151cc466fda0b7de53b19249ea5fdb5/v22.1/aws-dms.md#L26)):*
<details><summary><i>Previously, awoods187 (Andy Woods) wrote…</i></summary><blockquote>

isn't this 21.1.1? Or was the .0 sufficient?
</blockquote></details>

I was always under the assumption that the 0 release was fine, but I could write this in such a way to push end users to run the latest hotfix anyway. What are your thoughts?

**Review by devadvocado — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14482)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @otan)

___
*[`v22.1/aws-dms.md` line 26 at r1](https://reviewable.io/reviews/cockroachdb/docs/14482#-N6PXAphB_fR6B9b45lb:-N6TLWB28o6MQoGCaT75:b5d4qvg) ([raw file](https://github.com/cockroachdb/docs/blob/2ed345490151cc466fda0b7de53b19249ea5fdb5/v22.1/aws-dms.md#L26)):*
<details><summary><i>Previously, ajstorm (Adam Storm) wrote…</i></summary><blockquote>

I thought the .0 was fine too.  @otan?
</blockquote></details>

In the enablement, I just wrote 22.1.1. That may be causing the confusion here. Waiting for @otan to verify and will update slides if I made the mistake.

File: v21.2/aws-dms.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Ensure the following items are completed prior to starting this tutorial:
 
 - Configure a [source endpoint](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.html) in AWS pointing to your source database.
 - Configure a [replication instance](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_ReplicationInstance.html) in AWS.
-- Ensure you have a secure, publicly available [CockroachDB cluster](../cockroachcloud/create-a-serverless-cluster.html) running v21.2 or higher.
+- Ensure you have a secure, publicly available CockroachDB cluster running v21.2.13 or higher.
```
Comment (otan) [inline v21.2/aws-dms.md:26]: it doesnt technically haven't to be secure ... ;)

File: v21.2/aws-dms.md [hunk 2]
```diff
@@ -61,6 +64,7 @@ A database migration task, also known as a replication task, controls what data
 1. For the **Editing mode** radio button, keep **Wizard** selected.
 1. For the **Target table preparation mode**, select either **Truncate** or **Do nothing**. All other settings can remain as-is.
     <img src="{{ 'images/v21.2/aws-dms-task-settings.png' | relative_url }}" alt="AWS-DMS-Task-Settings" style="max-width:100%" />
+1. Manually create all schema objects in the target CockroachDB database. This step is required in order for the migration to populate data successfully.
```
Comment (otan) [inline v21.2/aws-dms.md:67]: would drop a hint that each table has to have a PRIMARY KEY though.

**Review by otan — APPROVED** [review]

**Review by awoods187 — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14482)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @awoods187, @devadvocado, @nickvigilante, and @otan)

___
*[`v21.2/aws-dms.md` line 67 at r2](https://reviewable.io/reviews/cockroachdb/docs/14482#-N6PW6zvnZ3S4pjUuxhb-r2-67:-N6k6sNf35-5zjQZTlK7:b32v57) ([raw file](https://github.com/cockroachdb/docs/blob/765241b6ee46e6faa2252901f4d262be9235c5eb/v21.2/aws-dms.md#L67)):*
<details><summary><i>Previously, otan (Oliver Tan) wrote…</i></summary><blockquote>

would drop a hint that each table has to have a PRIMARY KEY though.
</blockquote></details>

+1

**Review by nickvigilante — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14482)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @awoods187, @devadvocado, and @otan)

___
*[`v21.2/aws-dms.md` line 26 at r2](https://reviewable.io/reviews/cockroachdb/docs/14482#-N6PW6zvnZ3S4pjUuxhb-r2-26:-N6mnjW_6aq1lMj4iPo_:b-508g6w) ([raw file](https://github.com/cockroachdb/docs/blob/765241b6ee46e6faa2252901f4d262be9235c5eb/v21.2/aws-dms.md#L26)):*
<details><summary><i>Previously, otan (Oliver Tan) wrote…</i></summary><blockquote>

it doesnt technically haven't to be secure ... ;)
</blockquote></details>

:) I'm sure not, but in order to avoid a lawsuit, I'm gonna say it should be secure.
___
*[`v21.2/aws-dms.md` line 67 at r2](https://reviewable.io/reviews/cockroachdb/docs/14482#-N6PW6zvnZ3S4pjUuxhb-r2-67:-N6mngKkEYeQgeeXzdeW:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/765241b6ee46e6faa2252901f4d262be9235c5eb/v21.2/aws-dms.md#L67)):*
<details><summary><i>Previously, awoods187 (Andy Woods) wrote…</i></summary><blockquote>

+1
</blockquote></details>

Done.
___
*[`v22.1/aws-dms.md` line 26 at r1](https://reviewable.io/reviews/cockroachdb/docs/14482#-N6PXAphB_fR6B9b45lb:-N6mns9b-YUllt6Sw9hK:b-f8353v) ([raw file](https://github.com/cockroachdb/docs/blob/2ed345490151cc466fda0b7de53b19249ea5fdb5/v22.1/aws-dms.md#L26)):*
<details><summary><i>Previously, devadvocado (Kevin C) wrote…</i></summary><blockquote>

In the enablement, I just wrote 22.1.1. That may be causing the confusion here. Waiting for @otan to verify and will update slides if I made the mistake.
</blockquote></details>

@otan Waiting on you on this one.

**Review by devadvocado — APPROVED** [review]

**Review by otan — APPROVED** [review]

**Review by stbof — APPROVED** [review]

**Pull request merged**